mlflow/transformers.py

```diff
@@ -1800,6 +1807,52 @@ def _coerce_exploded_dict_to_single_dict(self, data):
         else:
             return data
 
+    def _parse_zero_shot_text_classifier_output_to_df(self, data):
+        """
+        Converts the output of sequences, labels, and scores to a Pandas DataFrame output.
+
+        Example input:
+
+        [{'sequence': 'My dog loves to eat spaghetti',
+          'labels': ['happy', 'sad'],
+          'scores': [0.9896970987319946, 0.010302911512553692]},
+         {'sequence': 'My dog hates going to the vet',
+          'labels': ['sad', 'happy'],
+          'scores': [0.957074761390686, 0.042925238609313965]}]
+
+        Output:
+
+        pd.DataFrame in a fully normalized (flattened) format with each sequence, label, and score
+        having a row entry.
+        For example, here is the DataFrame cast to a dictionary after conversion:
+
+        {'sequence': {0: 'My dog loves to eat spaghetti',
+          1: 'My dog loves to eat spaghetti',
+          2: 'My dog hates going to the vet',
+          3: 'My dog hates going to the vet'},
+         'label': {0: 'happy', 1: 'sad', 2: 'sad', 3: 'happy'},
+         'score': {0: 0.9896970987319946,
+          1: 0.010302911512553692,
+          2: 0.957074761390686,
+          3: 0.042925238609313965}}
```

Comment:
Can we replace this with what the dataframe looks like?

Reply:
great point. Changing to the `.to_string()` output format